src/components/MovieCard.vue

```diff
@@ -36,7 +38,7 @@ export default {
 @media (min-width: 0px) {
   .movie-card {
     width: 327px;
-    max-height: 400px;
+    height: 400px;
```

Comment:
I'm not sure about setting the height as const value. On smaller devices it would be taller than half of the device. What about height being calculated automatically based on the content inside + paddings top and bottom?